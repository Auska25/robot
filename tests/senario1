PLACE 0,0,NORTH
MOVE
MOVE
REPORT
LEFT
LEFT
MOVE
REPORT
QUIT

