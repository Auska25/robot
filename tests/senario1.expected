SYSTEM START
0,2,NORTH
0,1,SOUTH
SYSTEM EXIT
